docs/automated-checkout-services/device_services.html

```diff
@@ -887,32 +887,32 @@ <h3 id="controller-board-physical-device-functionality">Controller board physica
 <p>Therefore, if you have multiple serial devices plugged into your system, please manually edit the <code>docker-compose.yml</code> file to mount your <code>/dev/ttyACM_X_</code> appropriately. You may want to explore creating udev rules to enforce TTY consistency.</p>
 <h3 id="controller-board-apis">Controller board APIs</h3>
 <hr />
-<h4 id="put-httplocalhost48098apiv1devicenameds-controller-boardcommandlock1"><code>PUT</code>: <code>http://localhost:48098/api/v1/device/name/ds-controller-board/command/lock1</code></h4>
+<h4 id="put-httplocalhost48098apiv1devicenameds-controller-boardcommandlock1"><code>PUT</code>: <code>http://localhost:48098/api/v2/device/name/ds-controller-board/command/lock1</code></h4>
```

Comment:
is this html generated from a tool?  if not, could we make this long line as separate lines for more readable?
Similar thing applied to this whole html having a long line.

Reply:
@jim-wang-intel Yes, this was originally generated by a tool and should be regenerated.